api_testing/test_creating_resource.py

```diff
@@ -0,0 +1,11 @@
+from api_testing.credentials import CredentialsCreating
+
+
+def test_creating_resource(session, base_url):
+    creating_resource = session.post(base_url, json=CredentialsCreating.valid_data)
+    assert creating_resource.status_code == 201
```

Comment:
You could add checks for **userId**, **id**, **title** and **completed** response fields:

```
def test_creating_resource(session, base_url):
    res = session.post(base_url, json=CredentialsCreating.valid_data)
    assert res.status_code == 201
    r_json = res.json()
    assert r_json['userId'] == CredentialsCreating.valid_data['userId']
    assert r_json['id'] == CredentialsCreating.valid_data['id']
    assert r_json['title'] == CredentialsCreating.valid_data['title']
    assert r_json['completed'] == CredentialsCreating.valid_data['completed']
```